cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/network/compare/IssuesTest.java

```diff
@@ -69,17 +67,17 @@ private static Network createNetwork() {
         vl1.getBusBreakerView().newBus()
             .setId("B1")
             .add();
-        Generator g1 = vl1.newGenerator()
+        vl1.newGenerator()
             .setId("G1")
             .setBus("B1")
             .setMinP(0)
             .setMaxP(1)
             .setTargetP(1)
             .setTargetQ(0)
+            .useLocalRegulation(true)
             .setVoltageRegulatorOn(true)
             .setTargetV(400)
             .add();
-        assertEquals(g1.getRegulatingTerminal(), g1.getTerminal());
```

Comment:
You can keep this assert I think

Reply:
It is not necessary and can be deleted as useLocalRegulation has previously been set to true.

> Thank you for your great work! I have a few remarks but it is pretty much okay for me.
> Can you still explain to me why you set `targetQ` to 0.0 when it is `NaN` sometimes in `GeneratorXml`?
> Also why in checks, you now always check all the regulation values even if only `targetDeadband` has been changed?
> It is just for me to understand your choices!

I answered in conversations at the top

Reply:
Done. I kept the assert